documentation/objects/objects.md

```diff
@@ -17,47 +17,38 @@ weight: 1
 
 As the name suggests, the object classification workflow aims to classify full *objects*, based on object-level features and user annotations.
 An *object* in this context is a set of pixels that belong to the same instance.
-In order to do so, the workflow needs *segmentation* images besides the usual raw image data, that can e.g. be generated with the [Pixel Classification Workflow].
-Depending on the availability of these segmentation images, the user can choose between three flavors of object classification workflow, which differ by their input data:
+In order to do so, the workflow needs *segmentation* images besides the usual raw image data, which classify the pixels as either belonging to an object or not. These can be obtained e.g. using the [Pixel Classification Workflow].
```

Comment:
the term _segmentation_ is confusing here in my opinion (as we have two workflows, one with input segmentation, but we still refer to both of these inputs as segmentations...